infra-envs.yml

```diff
@@ -0,0 +1,15 @@
+- name: Test infra-envs
+  hosts: localhost
+  tasks:
+
+    - name: Create a infra-env
+      infra-envs:
+        state: present
+        name: testinfra
+        pull_secret: '<insert your pullsecret between the single quotes>'
+
+      register: registered_infraenvs
+
+    - name: Log registered_infraenvs Results
+      debug:
+        var: registered_infraenvs
```

Comment:
The playbook should probably include other use cases, i.e. list and delete.

Reply:
Yes, we have separate issues for list and delete. Will update as part of those issues.